lib/public/views/Logs/Logs.js

```diff
@@ -334,4 +315,33 @@ export default class Overview extends Observable {
             this.notify();
         }
     }
+
+    /**
+     * Set the title paramter in the model
+     * @param {String} title Received string from the view
+     * @returns {undefined}
+     */
+    setTitle(title) {
+        this.title = title;
+        this.notify();
+    }
+
+    /**
+     * Set the text paramter in the model
+     * @param {String} text Received string from the view
+     * @returns {undefined}
+     */
+    setText(text) {
+        this.text = text;
+        this.notify();
+    }
+
+    /**
+     * Sets all data related to the Logs to `NotAsked`.
+     * @returns {undefined}
+     */
+    clearLogs() {
```

Comment:
If the page is destroyed, shouldn't we also reset the paginiation options here?

Reply:
Good point - done